binaryaudit/dnf.py

```diff
@@ -133,7 +133,7 @@ def generate_abidiffs(key, source_dir, new_json_file, old_json_file, output_dir,
         with open("output_file", "w") as output_file:
             start_time = time.time()
             abipkgdiff, abipkgdiff_exit_code = run.run_command(command_list, None, output_file)
-            exec_time = time.time()-start_time
+            exec_time = (time.time()-start_time)*1000000
```

Comment:
Not related to this chane ;) I'd recommend to go by a separate patch and also switch to use `time.monotonic` instead.

Thanks